features/step_definitions/work_steps.rb

```diff
@@ -693,3 +693,25 @@
 Then /^the Remove Me As Chapter Co-Creator option should not be on the ([\d]+)(?:st|nd|rd|th) chapter$/ do |chapter_number|
   step %{I should not see "Remove Me As Chapter Co-Creator" within "ul#sortable_chapter_list > li:nth-of-type(#{chapter_number})"}
 end
+
+Then /^the share modal should contain ("|')(.*?)\1$/ do |quote, text|
```

Comment:
Instead of doing this, it might be good to set the test environment's `action_controller.default_url_options ` to `{ host: "http://www.example.com", port: nil }`, much like we do for Action Mailer: https://github.com/otwcode/otwarchive/blob/a18a4302b6f9e0a95f53dd0c2e7a54f05f4acedc/config/environments/test.rb#L55

We can probably even make things a bit more DRY by doing this:
```
  config.action_controller.default_url_options = { host: "http://www.example.com", port: nil }
  config.action_mailer.default_url_options = config.action_controller.default_url_options
```

(Thanks for the suggestion, redsummernight!)

Reply:
We might need to use `http://test.host` instead of `http://www.example.com` since Travis seems rather attached to the former.

Reply:
Thank you! ✨ 🧙 ✨  that is the magical incantation I was looking for. I've taken a stab with example.com in b9bbcd4610e2c724b0cc138bb41392f1832d0f11. If that doesn't work I'll try test.host.